cob_navigation_global/launch/amcl_node.xml

```diff
@@ -2,14 +2,16 @@
 <launch>
 
   <!-- combine front and rear scanner -->
-  <!--node pkg="topic_tools" type="relay" name="scan_front_relay" args="/base_laser_front/scan scan_combined"/>
-    <node pkg="topic_tools" type="relay" name="scan_rear_relay" args="/base_laser_rear/scan scan_combined"/-->
+  <node pkg="topic_tools" type="relay" name="scan_front_relay" args="/base_laser_front/scan scan_combined"/>
```

Comment:
I think we can get rid of the relays and use the scan_combined which is configurable per robot in cob_hardware_config. @ipa-mig: any reason to keep the relays?